packages/perseus/src/util/graphie.ts

```diff
@@ -1538,22 +1538,12 @@ const GraphUtils: any = {
             .join("");
     },
 
-    getDistance: function (point1, point2) {
+    getDistance: function (point1: Coord, point2: Coord): number {
         return kpoint.distanceToPoint(point1, point2);
     },
 
-    /**
-     * Round the given coordinates to a given snap value
-     * (e.g., nearest 0.2 increment)
-     */
-    snapCoord: function (coord, snap) {
```

Comment:
I'm really surprised this is unused. I wonder how we do coordinate snapping when dragging the movable points/lines around if this isn't used.  

No action required.

Reply:
I haven't dug into it too deeply, but it looked to me like the Interactive code was defining and using its own snap methods.